.travis.yml

```diff
@@ -22,14 +18,16 @@ env:
 install:
   - mkdir -p $GOPATH/bin
   - go get github.com/go-playground/overalls
-  - make deps
+  - go get github.com/goreleaser/goreleaser
```

Comment:
why not use `make deps` here?

Reply:
Because it caused `go mod vendor` to be called as part of deps and we don't use vendored mode in travis.